### PR TITLE
document and git-ignore the new keystore provided by beats

### LINKED ISSUE #559: Test and document keystore functionality from beats for APM Server
Beats merged a keystore (https://github.com/elastic/beats/pull/5687) for 6.2. We should make ourselves familiar with it and see if it "just works" for APM Server. If that is the case, we should document it. See https://github.com/elastic/beats/pull/6173.

This doesn't strictly need to happen before 6.2 release, but would be good to investigate at least.

### PR DESCRIPTION
**Pull request by jalvz**
fixes #559

### CODE REVIEW

**Comment by jalvz** [comment]
jenkins retest this please

**Comment by jalvz** [comment]
jenkins retest it please

File: docs/copied-from-beats/command-reference.asciidoc [hunk 1]
```diff
@@ -16,23 +16,22 @@
 
 :export-command-short-desc: Exports the configuration or index template to stdout
 :help-command-short-desc: Shows help for any command
+:keystore-command-short-desc: Manages the <<keystore,secrets keystore>>
 :modules-command-short-desc: Manages configured modules
 :run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
-
-ifeval::["{has_ml_jobs}"=="yes"]
```
Comment (simitt) [inline docs/copied-from-beats/command-reference.asciidoc]: Please don't remove this, there is a PR https://github.com/elastic/beats/pull/6184/files open in beats that introduces this distinction, as it doesn't make sense to describe ML here for APM.

Comment (jalvz) [reply docs/copied-from-beats/command-reference.asciidoc]: I copied over a fresh version of the command-reference that has an entry for the keystore.
Once that PR gets merged and copied over, we will have the right thing, no?

I think is better to wait for that PR to get merged first, than manually editing copied files.

**Comment by jalvz** [comment]
Yes, I tested and it works.

File: docs/copied-from-beats/command-reference.asciidoc (hunk 1 above)
Comment (simitt) [reply docs/copied-from-beats/command-reference.asciidoc]: As soon as this gets merged, it is in the official docs. 

Thus, I'd not suggest to merge in some documentation that is not supposed to show up for APM Server, but rather to copy over the part for the keystore as is and leave the other part of the file untouched for now.

**Comment by simitt** [comment]
As the https://github.com/elastic/beats/pull/6184/files is merged, could you please update this PR so we can merge it.

**Comment by jalvz** [comment]
I did already, the change up there in L387 seems to be consistent with the updated docs

**Pull request merged**